from PyQt6 import QtCore, QtGui, QtWidgets

class Ui_meet_the_team(object):
    def setupUi(self, meet_the_team):
        meet_the_team.setObjectName("meet_the_team")
        meet_the_team.resize(709, 817)
        meet_the_team.setStyleSheet("background-image: url(static/logo3.jpg);"
                                    "background-repeat: no-repeat;"
                                    "background-position: center;")

        self.member1 = QtWidgets.QLabel(parent=meet_the_team)
        self.member1.setGeometry(QtCore.QRect(70, 10, 51, 16))
        font = QtGui.QFont()
        font.setFamily("Rockwell")
        font.setPointSize(10)
        font.setBold(True)
        font.setUnderline(True)
        self.member1.setFont(font)
        self.member1.setStyleSheet("color: rgb(161, 129, 200);")
        self.member1.setObjectName("member1")

        self.member2 = QtWidgets.QLabel(parent=meet_the_team)
        self.member2.setGeometry(QtCore.QRect(270, 10, 31, 16))
        font = QtGui.QFont()
        font.setFamily("Rockwell")
        font.setPointSize(10)
        font.setBold(True)
        font.setUnderline(True)
        self.member2.setFont(font)
        self.member2.setStyleSheet("color: rgb(161, 129, 200);")
        self.member2.setObjectName("member2")

        self.member3 = QtWidgets.QLabel(parent=meet_the_team)
        self.member3.setGeometry(QtCore.QRect(430, 10, 41, 16))
        font = QtGui.QFont()
        font.setFamily("Rockwell")
        font.setPointSize(10)
        font.setBold(True)
        font.setUnderline(True)
        self.member3.setFont(font)
        self.member3.setStyleSheet("color: rgb(161, 129, 200);")
        self.member3.setObjectName("member3")

        self.member4 = QtWidgets.QLabel(parent=meet_the_team)
        self.member4.setGeometry(QtCore.QRect(600, 10, 41, 16))
        font = QtGui.QFont()
        font.setFamily("Rockwell")
        font.setPointSize(10)
        font.setBold(True)
        font.setUnderline(True)
        self.member4.setFont(font)
        self.member4.setStyleSheet("color: rgb(161, 129, 200);")
        self.member4.setObjectName("member4")

        self.nailslbl = QtWidgets.QLabel(parent=meet_the_team)
        self.nailslbl.setGeometry(QtCore.QRect(590, 450, 49, 16))
        font = QtGui.QFont()
        font.setFamily("Bodoni MT")
        font.setPointSize(10)
        font.setBold(True)
        self.nailslbl.setFont(font)
        self.nailslbl.setStyleSheet("color: rgb(170, 0, 255);")
        self.nailslbl.setObjectName("nailslbl")

        self.hairlbl = QtWidgets.QLabel(parent=meet_the_team)
        self.hairlbl.setGeometry(QtCore.QRect(330, 310, 49, 16))
        font = QtGui.QFont()
        font.setFamily("Bodoni MT")
        font.setPointSize(11)
        font.setBold(True)
        self.hairlbl.setFont(font)
        self.hairlbl.setStyleSheet("color: rgb(170, 0, 255);")
        self.hairlbl.setObjectName("hairlbl")

        self.cosmeticslbl = QtWidgets.QLabel(parent=meet_the_team)
        self.cosmeticslbl.setGeometry(QtCore.QRect(50, 470, 81, 16))
        font = QtGui.QFont()
        font.setFamily("Bodoni MT")
        font.setPointSize(11)
        font.setBold(True)
        self.cosmeticslbl.setFont(font)
        self.cosmeticslbl.setStyleSheet("color: rgb(170, 0, 255);")
        self.cosmeticslbl.setObjectName("cosmeticslbl")

        # Updated paths
        self.member1_photo = QtWidgets.QLabel(parent=meet_the_team)
        self.member1_photo.setGeometry(QtCore.QRect(20, 30, 161, 221))
        self.member1_photo.setStyleSheet("border: 2px solid black; border-radius: 10px;")
        self.member1_photo.setText("")
        self.member1_photo.setPixmap(QtGui.QPixmap("static/IMG-20240730-WA0116.jpg"))
        self.member1_photo.setObjectName("member1_photo")

        self.member2_photo = QtWidgets.QLabel(parent=meet_the_team)
        self.member2_photo.setGeometry(QtCore.QRect(210, 30, 141, 221))
        self.member2_photo.setStyleSheet("border: 2px solid black; border-radius: 10px;")
        self.member2_photo.setText("")
        self.member2_photo.setPixmap(QtGui.QPixmap("static/IMG-20240730-WA0114.jpg"))
        self.member2_photo.setObjectName("member2_photo")

        self.member3_photo = QtWidgets.QLabel(parent=meet_the_team)
        self.member3_photo.setGeometry(QtCore.QRect(380, 30, 141, 221))
        self.member3_photo.setStyleSheet("border: 2px solid black; border-radius: 10px;")
        self.member3_photo.setText("")
        self.member3_photo.setPixmap(QtGui.QPixmap("static/IMG-20240730-WA0115.jpg"))
        self.member3_photo.setObjectName("member3_photo")

        self.member4_photo = QtWidgets.QLabel(parent=meet_the_team)
        self.member4_photo.setGeometry(QtCore.QRect(548, 30, 141, 221))
        self.member4_photo.setStyleSheet("border: 2px solid black; border-radius: 10px;")
        self.member4_photo.setText("")
        self.member4_photo.setPixmap(QtGui.QPixmap("static/IMG-20240730-WA0117.jpg"))
        self.member4_photo.setObjectName("member4_photo")

        self.member1_photo_2 = QtWidgets.QLabel(parent=meet_the_team)
        self.member1_photo_2.setGeometry(QtCore.QRect(300, 330, 101, 131))
        self.member1_photo_2.setStyleSheet("border: 2px solid black; border-radius: 10px;")
        self.member1_photo_2.setText("")
        self.member1_photo_2.setPixmap(QtGui.QPixmap("static/IMG-20240730-WA0116.jpg"))
        self.member1_photo_2.setObjectName("member1_photo_2")

        self.member3_photo_2 = QtWidgets.QLabel(parent=meet_the_team)
        self.member3_photo_2.setGeometry(QtCore.QRect(300, 470, 101, 131))
        self.member3_photo_2.setStyleSheet("border: 2px solid black; border-radius: 10px;")
        self.member3_photo_2.setText("")
        self.member3_photo_2.setPixmap(QtGui.QPixmap("static/IMG-20240730-WA0115.jpg"))
        self.member3_photo_2.setObjectName("member3_photo_2")

        self.member4_photo_2 = QtWidgets.QLabel(parent=meet_the_team)
        self.member4_photo_2.setGeometry(QtCore.QRect(300, 610, 101, 141))
        self.member4_photo_2.setStyleSheet("border: 2px solid black; border-radius: 10px;")
        self.member4_photo_2.setText("")
        self.member4_photo_2.setPixmap(QtGui.QPixmap("static/IMG-20240730-WA0117.jpg"))
        self.member4_photo_2.setObjectName("member4_photo_2")

        self.member1_photo_3 = QtWidgets.QLabel(parent=meet_the_team)
        self.member1_photo_3.setGeometry(QtCore.QRect(560, 470, 101, 131))
        self.member1_photo_3.setStyleSheet("border: 2px solid black; border-radius: 10px;")
        self.member1_photo_3.setText("")
        self.member1_photo_3.setPixmap(QtGui.QPixmap("static/IMG-20240730-WA0116.jpg"))
        self.member1_photo_3.setObjectName("member1_photo_3")

        self.member3_photo_3 = QtWidgets.QLabel(parent=meet_the_team)
        self.member3_photo_3.setGeometry(QtCore.QRect(560, 620, 101, 131))
        self.member3_photo_3.setStyleSheet("border: 2px solid black; border-radius: 10px;")
        self.member3_photo_3.setText("")
        self.member3_photo_3.setPixmap(QtGui.QPixmap("static/IMG-20240730-WA0115.jpg"))
        self.member3_photo_3.setObjectName("member3_photo_3")

        self.member2_photo_2 = QtWidgets.QLabel(parent=meet_the_team)
        self.member2_photo_2.setGeometry(QtCore.QRect(40, 490, 101, 131))
        self.member2_photo_2.setStyleSheet("border: 2px solid black; border-radius: 10px;")
        self.member2_photo_2.setText("")
        self.member2_photo_2.setPixmap(QtGui.QPixmap("static/IMG-20240730-WA0114.jpg"))
        self.member2_photo_2.setObjectName("member2_photo_2")

        self.bookApptBtn = QtWidgets.QPushButton(parent=meet_the_team)
        self.bookApptBtn.setGeometry(QtCore.QRect(10, 730, 141, 24))
        self.bookApptBtn.setStyleSheet("color: rgb(170, 0, 255);")
        self.bookApptBtn.setObjectName("bookApptBtn")

        self.backBtn = QtWidgets.QPushButton(parent=meet_the_team)
        self.backBtn.setGeometry(QtCore.QRect(40, 760, 75, 24))
        self.backBtn.setStyleSheet("color: rgb(170, 0, 255);")
        self.backBtn.setObjectName("backBtn")

        self.retranslateUi(meet_the_team)
        QtCore.QMetaObject.connectSlotsByName(meet_the_team)

    def retranslateUi(self, meet_the_team):
        _translate = QtCore.QCoreApplication.translate
        meet_the_team.setWindowTitle(_translate("meet_the_team", "Meet The Team"))
        self.member1.setText(_translate("meet_the_team", "Roxana"))
        self.member2.setText(_translate("meet_the_team", "Eva"))
        self.member3.setText(_translate("meet_the_team", "Maria"))
        self.member4.setText(_translate("meet_the_team", "Daria"))
        self.nailslbl.setText(_translate("meet_the_team", "NAILS"))
        self.hairlbl.setText(_translate("meet_the_team", "HAIR"))
        self.cosmeticslbl.setText(_translate("meet_the_team", "COSMETICS"))
        self.bookApptBtn.setText(_translate("meet_the_team", "Book an appointment"))
        self.backBtn.setText(_translate("meet_the_team", "Go Back"))